Academic-Honesty-Policy.md

```diff
@@ -1,3 +1,7 @@
+# Academic Honesty Policy
+
+## 

```

Comment:
This one does not need it, also with the way you have it it would only show the title on the chat anyways.